do

```diff
@@ -15,13 +15,21 @@ version() {
 help_kubeconform="Run helm kubeconform"
 kubeconform() {
     check-helm
-
     install-plugin kubeconform https://github.com/jtyr/kubeconform-helm
 
-    helm kubeconform --ignore-missing-schema --verbose --summary --strict "$@" \
+    # Add schemas not included in any catalogs
+    schemas_dir="./target/schemas"
+    mkdir -p "${schemas_dir}"
+    cd "${schemas_dir}"
+    curl -s https://raw.githubusercontent.com/yannh/kubeconform/master/scripts/openapi2jsonschema.py | \
+        python3 - https://raw.githubusercontent.com/grafana/tempo-operator/refs/heads/main/config/crd/bases/tempo.grafana.com_tempomonolithics.yaml
+    cd -
```

Comment:
Unfortunately, there's no catalog with the CRDs schema for the Tempo operator, so we'll just generate one ourselves using the provided script from `kubeconform`. Testing: https://app.circleci.com/pipelines/github/CircleCI-Public/circleci-server-monitoring-reference/476/workflows/3db1c827-b679-4544-ba0f-196c6cdcd26b/jobs/2281?invite=true#step-105-6511_56